app/src/main/java/com/stl/skipthelibrary/LoginActivity.java

```diff
@@ -0,0 +1,38 @@
+package com.stl.skipthelibrary;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.Toast;
+
+public class LoginActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login);
+    }
+
+
+    public void signInOnClick(View view) {
+        String email = ((EditText) findViewById(R.id.UsernameEditText)).getText().toString();
```

Comment:
Its common practice to grab all the UI elements in the onCreate and have the variables as attributes of the activity rather than assigning them within a method. I'd probably do that here
```suggestion
        String email = usernameText.getText().toString();
```